src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1146,10 +1147,12 @@ protected override void GetItem(string path)
                         }
                     }
                 }
+#endif
 
                 FileSystemInfo result = GetFileSystemItem(path, ref isContainer, false);
                 if (result != null)
                 {
+#if !UNIX
                     // If we want to retrieve the file streams, retrieve them.
                     if (retrieveStreams)
```

Comment:
The alternate streams can be specified using `-Stream` or an inline stream syntax `"file::stream"`, the check of the inline stream syntax is done in 7 places in `FileSystemProvider.cs` but some of them are not within `#if !UNIX` block.